src/AvTranscoder/transcoder/StreamTranscoder.cpp

```diff
@@ -550,6 +550,10 @@ bool StreamTranscoder::processTranscode()
         LOG_DEBUG("Convert")
         _transform->convert(*_filteredData, *_transformedData);
 
+        // free the filtered AVFrame now it has been converted into another one
```

Comment:
Could you add a warning in the doc of the FilterGraph::process method about that ?